controllers/k8ssandra/medusa_reconciler.go

```diff
@@ -137,3 +148,64 @@ func (r *K8ssandraClusterReconciler) reconcileMedusaConfigMap(
 	logger.Info("Medusa ConfigMap successfully reconciled")
 	return result.Continue()
 }
+
+func (r *K8ssandraClusterReconciler) reconcileMedusaStandalone(
```

Comment:
This doesn't make sense, at least not here in the K8ssandraClusterReconciler. The motivation of having a stand alone Medusa deployment is that the remote storage can be queried when there is no K8ssandraCluster. 

The work for this should happen as part of #495 which I have started but not gotten too far yet.

Reply:
That's not exactly what I had understood. The motivation was to be able to communicate with the Medusa storage backend even if no Cassandra pods were up yet, in order to perform a restore on the very first start of these pods.
A K8ssandraCluster object is still our starting point IMO. It's where we'll define the name of the backup we want to restore, as well as the medusa backend settings.